packages/web-components/fast-foundation/src/calendar/calendar.ts

```diff
@@ -0,0 +1,339 @@
+    selected?: boolean;
+};
+
+/**
+ * Calendar component
+ * @public
+ */
+export class Calendar extends FoundationElement {
+    /**
+     * date formatter utitlity for getting localized strings
+     */
+    @observable
+    public dateFormatter: DateFormatter = new DateFormatter();
+
+    /**
+     * String repesentation of the full locale including market, calendar type and numbering system
+     * @public
+     */
+    @attr
+    public locale: string = "en-US";
+    private localeChanged(): void {
+        this.dateFormatter.locale = this.locale;
+    }
+
+    /**
+     * Month to display
+     * @public
+     */
+    @attr({ converter: nullableNumberConverter })
+    public month: number = new Date().getMonth() + 1;
+
+    /**
+     * Year of the month to display
+     * @public
+     */
+    @attr({ converter: nullableNumberConverter })
+    public year: number = new Date().getFullYear();
+
+    /**
+     * Format style for the day
+     * @public
+     */
+    @attr({ attribute: "day-format" })
+    public dayFormat: DayFormat = "numeric";
+    private dayFormatChanged(): void {
+        this.dateFormatter.dayFormat = this.dayFormat;
+    }
+
+    /**
+     * Format style for the week day labels
+     * @public
+     */
+    @attr({ attribute: "weekday-format" })
+    public weekdayFormat: WeekdayFormat = "short";
+    private weekdayFormatChanged(): void {
+        this.dateFormatter.weekdayFormat = this.weekdayFormat;
+    }
+
+    /**
+     * Format style for the month label
+     * @public
+     */
+    @attr({ attribute: "month-format" })
+    public monthFormat: MonthFormat = "long";
+    private monthFormatChanged(): void {
+        this.dateFormatter.monthFormat = this.monthFormat;
+    }
+
+    /**
+     * Format style for the year used in the title
+     * @public
+     */
+    @attr({ attribute: "year-format" })
+    public yearFormat: YearFormat = "numeric";
+    private yearFormatChanged(): void {
+        this.dateFormatter.yearFormat = this.yearFormat;
+    }
+
+    /**
+     * Minimum number of weeks to show for the month
+     * This can be used to normalize the calendar view
+     *  when changing or across multiple calendars
+     * @public
+     */
+    @attr({ attribute: "min-weeks", converter: nullableNumberConverter })
+    public minWeeks: number = 0;
+
+    /**
+     * A list of dates that should be shown as disabled
+     * @public
+     */
+    @attr({ attribute: "disabled-dates" })
+    public disabledDates: string = "";
+
+    /**
+     * A list of dates that should be shown as highlighted
+     * @public
+     */
+    @attr({ attribute: "selected-dates" })
+    public selectedDates: string = "";
+
+    /**
+     * The number of miliseconds in a day
+     * @internal
+     */
+    private oneDayInMs: number = 86400000;
+
+    /**
+     * An evaluation of whether or not the calendar is rendered as RTL
+     * @internal
+     */
+    public isRTL(locale: string = this.locale): boolean {
```

Comment:
Where is this used?

Reply:
It was just used in testing. Removed this.